### DISCUSSION TITLE
在支持资源包与地图下载时，顺带把光影下载功能也支持了

### DISCUSSION THREAD

**Discussion by shaoxiu** [Meloong-Git/PCL#443] [提问] [answered]
目前Curseforge分离了资源包板块与光影板块，因此理论上也应该可以顺带做了( ? )最大的问题应该是光影API的选择。Curseforge上的所有光影都是在optifine环境下开发的，仅有部分支持iris。建议点开此板块时先检测玩家是否安装optifine或iris，若检测到iris，则以弹窗或其它形式告知用户它不一定支持所有光影，从而避免用户乱甩锅现象发生。

**Comment by WForst-Breeze** [comment, marked as answer]
已有相关issue提及，由于下载源问题，不会制作，见 #396
